students/Lina/Lesson9/donor_models.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python3
+#------------------------------------------------------
+# Lesson 9 - Assignment 8: Mailroom - Object Oriented
+#
+#            donor_models.py
+#               class Donor
+#               class DonorCollection
+#------------------------------------------------------
+
+from datetime import date
+from operator import itemgetter
+
+# Donor class contains information of a single donor
+class Donor:
+    def __init__(self, donor_name, donor_details=None):
+        self.full_name = donor_name
+        if donor_details == None:
+            details = self.lookup_donor(donor_name)
+            if details:
+                self.details = details
+            else:
+                self.details = {"gift_amounts" : [0]}
+        else:
+            self.details = donor_details
+
+    @classmethod
+    def lookup_donor(cls, name):
+        return DonorCollection().lookup_donor(name)
```

Comment:
this method does not belong here - remember this class is all about a single donor, looking up a donor deals with looking through a collection of donors and belongs in DonorCollection class